build.bash

```diff
@@ -148,25 +145,21 @@ elif [ "$1" == "rpi64" ]; then
   hw_platform="pi-raspios64beta"
   echo_process "Hardware platform: Raspberry Pi (rpi64) - BETA -"
 
-elif [ "$1" == "pine64" ]; then
-  hw_platform="pine64-xenial"
-  echo_process "Hardware platform: Pine A64 (pine64)"
-
 elif [ "$1" == "local-test" ]; then
   echo_process "Preparing local system for installation"
   cp ./build-image/first-boot.bash /boot/first-boot.bash
   cp ./build-image/webif.bash /boot/webif.bash
   cp ./build-image/openhabian.conf /boot/openhabian.conf
-  cp ./build-image/rc.local /etc/rc.local
+  cp ./build-image/openhabian-installer.service /etc/systemd/system/
+  ln -sf /etc/systemd/system/openhabian-installer.service /etc/systemd/system/multi-user.target.wants/openhabian-installer.service
+  rm -f /opt/openHABian-install-successful
+  rm -f /opt/openHABian-install-inprogress
   sed -i -e '1r functions/helpers.bash' /boot/first-boot.bash # Add platform identification
   # Use local filesystem's version of openHABian
-  sed -i 's|if \[ -d /opt/openhabian/ \]; then cd /opt && rm -rf /opt/openhabian/; fi||' /boot/first-boot.bash
-  # shellcheck disable=SC2016
-  sed -i 's|if git clone -q -b "$clonebranch" "$repositoryurl" /opt/openhabian; then echo "OK"; else echo "FAILED"; fail_inprogress; fi||' /boot/first-boot.bash
+  sed -i 's|openhabian_update|true|' /boot/first-boot.bash
   chmod +x /boot/first-boot.bash
   chmod +x /boot/webif.bash
-  chmod +x /etc/rc.local
-  echo_process "Local system ready for installation test. Run script /etc/rc.local or reboot to initiate"
+  echo_process "Local system ready for installation test. Run systemctl start openhabian-installer or reboot to initiate"
```

Comment:
I would change "Run systemctl start openhabian-installer or ..." to "Run 'systemctl start openhabian-installer.service' to make it clear what to run.